pkg/proxy/proxy_community_test.go

```diff
@@ -287,73 +287,103 @@ func (s *TestProxySuite) checkProxyCommunityOK(fakeApp *fake.ProxyFakeApp, p *Pr
 			},
 			// Given user alice exists
 			// And   alice owns a private workspace
-			// When  alice requests the list of pods in a non existing namespace in alice's workspace
-			// Then  the proxy does NOT forward the request
-			// And   the proxy rejects the call with 403 Forbidden
-			"plain http request as owner to not existing namespace in private workspace": {
-				ProxyRequestMethod:          "GET",
-				ProxyRequestHeaders:         map[string][]string{"Authorization": {"Bearer " + s.token(alice)}},
-				ExpectedProxyResponseStatus: http.StatusForbidden,
-				RequestPath:                 podsInNamespaceRequestURL("alice-private", "not-existing"),
-				ExpectedResponse:            "invalid workspace request: access to namespace 'not-existing' in workspace 'alice-private' is forbidden",
+			// When  alice requests the list of pods in a namespace which does not belong to the alice's workspace
+			// Then  the proxy does forward the request anyway.
+			// It's not up to the proxy to check permissions on the specific namespace.
+			// The target API server will reject the request if the user does not have permissions to access the namespace.
+			// Here the request is successful because the underlying mock target cluster API server returns OK
+			"plain http request as owner to namespace outside of private workspace": {
```

Comment:
I'm a bit confused as to what does 'owner' refer to here in the test description. To me, there is not much difference in the two statements below, wrt to the test:
`plain http request as owner to namespace outside of private workspace` v/s
`plain http request to namespace outside of private workspace`. 
So either I'm missing the significance of `owner` here and we should add it here (maybe in the comments?) or we can make the test case description simpler and remove owner.
Let me know what you think

Reply:
Replaced `owner` by `permitted user`.